doc/source/names_wordlist.txt

```diff
@@ -0,0 +1,1652 @@
+Ferroni
+Haleemur
+harshul
+Hassan
+Shamim
+iamsimha
+Iulius
+Nazarov
+jackieleng
+Reback
+Crist
+Jevnik
+Liekezer
+Zwinck
+Erenrich
+Joris
+Bossche
+Howes
+Brandys
+Kamil
+Sindi
+Ka
+Wo
+Shedden
+Kernc
+Brucher
+Roos
+Scherer
+Mortada
+Mehyar
+mpuels
+Haseeb
+Tariq
+Bonnotte
+Virtanen
+Mestemaker
+Pawel
+Kordek
+Battiston
+pijucha
+Jucha
+priyankjain
+Nimmi
+Gieseke
+Keyes
+Sahil
+Dua
+Sanjiv
+Lobo
+Sašo
+Stanovnik
+Heide
+sinhrks
+Sinhrks
+Kappel
+Choi
+Sudarshan
+Konge
+Caswell
+Augspurger
+Uwe
+Hoffmann
+wcwagner
+Xiang
+Zhang
+Yadunandan
+Yaroslav
+Halchenko
+YG
+Riku
+Yuichiro
+Kaneko
+yui
+knk
+zhangjinjie
+znmean
+颜发才
+Yan
+Facai
+Fiore
+Gartland
+Bastiaan
+Benoît
+Vinot
+Fustin
+Freitas
+Ter
+Livschitz
+Gábor
+Lipták
+Hassan
+Kibirige
+Iblis
+Saeta
+Pérez
+Wolosonovich
+Reback
+Jevnik
+Joris
+Bossche
+Storck
+Ka
+Wo
+Shedden
+Kieran
+O'Mahony
+Lababidi
+Maoyuan
+Liu
+Wittmann
+MaxU
+Roos
+Droettboom
+Eubank
+Bonnotte
+Virtanen
+Battiston
+Prabhjot
+Singh
+Augspurger
+Aiyong
+Winand
+Xbar
+Yan
+Facai
+adneu
+ajenkins
+cargometrics
+behzad
+nouri
+chinskiy
+gfyoung
+jeps
+jonaslb
+kotrfa
+nileracecrew
+onesandzeroes
+sinhrks
+tsdlovell
+Alekseyev
+Rosenfeld
+Anthonios
+Partheniou
+Sipos
+Carroux
+Aycock
+Scanlin
+Da
+Dorozhko
+O'Donovan
+Cleary
+Gianluca
+Jeffries
+Horel
+Schwabacher
+Deschenes
+Reback
+Jevnik
+Fremlin
+Hoersch
+Joris
+Bossche
+Joris
+Vankerschaver
+Ka
+Wo
+Keming
+Zhang
+Shedden
+Farrugia
+Lurie
+Roos
+Mayank
+Asthana
+Mortada
+Mehyar
+Moussa
+Taifi
+Navreet
+Bonnotte
+Reiners
+Gura
+Battiston
+Carnevale
+Rinoc
+Rishipuri
+Sangmin
+Lasley
+Sereger
+Seabold
+Thierry
+Moisan
+Caswell
+Augspurger
+Hauck
+Varun
+Yoong
+Kang
+Lim
+Yoshiki
+Vázquez
+Baeza
+Joong
+Younggun
+Yuval
+Langer
+argunov
+behzad
+nouri
+boombard
+brian
+pantano
+chromy
+daniel
+dgram
+gfyoung
+hcontrast
+jfoo
+kaustuv
+deolal
+llllllllll
+ranarag
+rockg
+scls
+seales
+sinhrks
+srib
+surveymedia
+tworec
+Drozd
+Anthonios
+Partheniou
+Berendt
+Piersall
+Hamed
+Saljooghinejad
+Iblis
+Deschenes
+Reback
+Callin
+Joris
+Bossche
+Ka
+Wo
+Loïc
+Séguin
+Luo
+Yicheng
+Magnus
+Jöud
+Leonhardt
+Roos
+Bonnotte
+Pastafarianist
+Chong
+Schaf
+Philipp
+deCarvalho
+Khomenko
+Rémy
+Léone
+Thierry
+Moisan
+Augspurger
+Varun
+Hoffmann
+Winterflower
+Younggun
+ajcr
+azuranski
+behzad
+nouri
+cel
+emilydolson
+hironow
+lexual
+llllllllll
+rockg
+silentquasar
+sinhrks
+taeold
+unparseable
+Rothberg
+Bedini
+Rosenfeld
+Anthonios
+Partheniou
+Artemy
+Kolchinsky
+Willers
+Gohlke
+Clearfield
+Ringwalt
+Cottrell
+Gagne
+Schettino
+Panfilov
+Araujo
+Gianluca
+Poulin
+Nisar
+Henriksen
+Hoegen
+Jaidev
+Deshpande
+Swails
+Reback
+Buyl
+Joris
+Bossche
+Joris
+Vankerschaver
+Julien
+Danjou
+Ka
+Wo
+Kehoe
+Jordahl
+Shedden
+Buitinck
+Gambogi
+Savoie
+Roos
+D'Agostino
+Mortada
+Mehyar
+Eubank
+Nipun
+Batra
+Ondřej
+Čertík
+Pratap
+Vardhan
+Rafal
+Skolasinski
+Rinoc
+Gieseke
+Safia
+Abdalla
+Saumitra
+Shahapure
+Pölsterl
+Rubbert
+Sinhrks
+Siu
+Kwan
+Seabold
+Carrucciu
+Hoyer
+Pascoe
```

Comment:
these look like the names of our contributors? can we simply generate this?

something like: `git log --format='%an' | sort|uniq` (I guess would have to split on whitespace as well)

Reply:
The spelling extension checks every single file for text/docstrings so the names of the contributors needed to be whitelisted in order to be ignored by the spellchecker.

How would you advise on generating this? The reason why I'm asking is that the spelling extension uses `path.join` on anything that you pass as a wordlist (one or two files) so we only get an exception when trying to use let's say a list of names.

I suppose we could run the command and write it to file on every run? Perhaps using something like a tempfile to get all those names and use that file on every run?  

Also, I have encountered a few issues when decoding and splitting every name - the spelling extension makes some weird word divisions at times, when I try to decode and split the list  Yan Facai name will be split like this: `['颜发才（Yan', 'Facai）']` since the wordlist expects one line per word, then this name will always be marked as wrong due to how the split divided the name.  I suppose using regex could help with this matter but not sure.